CHANGELOG.md

```diff
@@ -28,7 +28,7 @@ scheme are considered to be bugs.
 
 ### Fixed
 
-* Your contribution here.
+* [#368](https://github.com/intridea/hashie/pull/368): hashie/mash can be required alone, require it's dependencies - [@jrafanie](https://github.com/jrafanie/)
```

Comment:
Also add a period at the end of this, quote `hashie/mash` and remove the trailing slash from `jrafanie/` ;)